src/image_intel.cpp

```diff
@@ -40,24 +44,102 @@ static void setBootAddress(size_t address)
     std::ignore = exec("fw_setenv bootcmd bootm %08x", address);
 }
 
+using FileSystem = std::string;
+using MountPoint = std::string;
+using MountPoints = std::vector<std::pair<FileSystem, MountPoint>>;
+
+static MountPoints getMountPoints()
+{
+    /* This regexp gives MTD partitions and their mount points.
+     * For example:
+     *   mtd:rwfs /tmp/.rwfs jffs2 rw,sync,relatime 0 0
+     *   mtd:sofs /var/sofs jffs2 rw,sync,relatime 0 0
+     * give two pairs:
+     *   {'mtd:rwfs', '/tmp/.rwfs'},
+     *   {'mtd:sofs', '/var/sofs'}
+     */
+    static const std::regex MTDPartsTmpl("^(mtd:\\w+)\\s+([^\\s]+)\\s+.*$");
```

Comment:
Is the trailing `\\s+.*$` really needed? What for?

Reply:
For a strictness. :)
The regex covers the whole line.